helix-term/src/commands/typed.rs

```diff
@@ -1066,30 +1066,26 @@ fn reflow(
     args: &[Cow<str>],
     _event: PromptEvent,
 ) -> anyhow::Result<()> {
+    let cfg_text_width: usize = cx.editor.config().text_width;
```

Comment:
I wonder if there is a better way rather than loading this all the time even when we don't need it, maybe the `current!` need to be changed to support config, but also the issue with config is that it is a function, `config().text_width` might seemed weird I think.

Reply:
> all the time even when we don't need it

What do you mean?

Reply:
`cfg_text_width` is only used within the last `.unwrap_or(cfg_text_width)` as fallback, so I don't know if we want to always load it. Sorry for the late reply.